app/services/katello/candlepin/consumer.rb

```diff
@@ -6,6 +6,7 @@ class Consumer
       ENTITLEMENTS_VALID = 'valid'.freeze
       ENTITLEMENTS_PARTIAL = 'partial'.freeze
       ENTITLEMENTS_INVALID = 'invalid'.freeze
+      UNSUBSCRIBED_HYPERVISOR = 'unsubscribed_hypervisor'.freeze
```

Comment:
I don't think this is needed?  The check should just be in the subscription_status